Tests/APITesters/AllAPITests/SwiftAPITester/VirtualCurrencyAPI.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  VirtualCurrencyAPI.swift
+//  SwiftAPITester
+//
+//  Created by Will Taylor on 2/28/25.
+//
+
+import Foundation
+import RevenueCat
+
+func checkVirtualCurrency(virtualCurrency: VirtualCurrency) {
+    let balance: Int = virtualCurrency.balance
+    let name: String = virtualCurrency.name
+    let code: String = virtualCurrency.code
+    let serverDescription: String? = virtualCurrency.serverDescription
```

Comment:
Probably not too important and for sure it's not something only in this API test, but I've just realized that doing this does not cover from us changing the `serverDescription` to nonnullable 🤔 
I guess we could add something like this:
```suggestion
    let serverDescription: String? = virtualCurrency.serverDescription
    var serverDescriptionAlt = virtualCurrency.serverDescription
    serverDescriptionAlt = ""
    serverDescriptionAlt = nil
```
If we where to change its type or nullability, one of the two lines would complain